docs/build-parachains-rococo.md

```diff
@@ -0,0 +1,108 @@
+---
+id: build-parachains-rococo
+title: Building Parachains on Rococo
+sidebar_label: Building Parachains on Rococo
+---
+
+[Rococo](https://github.com/paritytech/cumulus#rococo-crown) is a Polkadot testnet built for testing
+parachains. Rococo utilizes Cumulus and HRMP (Horizontal Relay-routed Message Passing) in order to
+send transfers and messages between parachains and the Relay Chain. Every message is sent to the
+Relay Chain, then from the Relay Chain to the desired parachain. Rococo currently runs parachains
+Tick, Trick and Track.
```

Comment:
"runs three test parachains (Tick Trick and Track), as well as several externally developed parachains.

Reply:
since there are more listed below..